cellprofiler/setting.py

```diff
@@ -1709,12 +1709,24 @@ def shape(self, value):
 
     @property
     def size(self):
-        return int(self.value_text.split(",")[1])
+        _, size = self.value_text.split(",")
```

Comment:
Additionally, we can't set `value_text` to `None` because it contains the structuring element shape information.